src/scss/layout/_title.scss

```diff
@@ -8,14 +8,14 @@
 
   align-self: center;
   grid-area: title;
-  margin: 0;
   position: relative;
 
+  margin: -50px 0 0 0; // needs a breakpoint for smaller screens
```

Comment:
I'm not sure what this is for, but `50px` feels like a magic number we should clarify, and not obviously related to anything else in our sizing. What's the goal here?

Reply:
@mirisuzanne this is to bump the "we create" part of the heading past the top of the image next to it.